### DISCUSSION TITLE
Configurator feature to write the config files directly to the proper places

### DISCUSSION THREAD

**Discussion by lagnat** [mriscoc/Ender3V2S1#925] [Ideas]
I was imagining that one could invoke the configurator like this: `python Configurator.pyw ../Ender3V2S1/` and upon pressing `Generate`, the files would get generated and placed in the appropriate places.

**Comment by mriscoc** [comment]
Something like this will create the configuration files directly and you can use a symlink to write into the project folder:
![image](https://github.com/mriscoc/Ender3V2S1/assets/2745567/f6816e3a-287a-40ad-b739-7e04d074d2d6)
